package.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "todolist",
+    "version": "1.0.0",
+    "description": "> It is a Todo list app build using JavaScript.",
+    "private": true,
+    "scripts": {
+        "start": "webpack-dev-server --open"
```

Comment:
Opps, we are missing the necessary scripts;
 We should be able to run common commands like `build`, `development` should be in `package.json` inside \`scripts'. Here is a basic example ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)) in our own case here, we are missing it so I can't run `npm run build ` to generate `dist folder`, I should be able to run `npm run build` to generate `dist folder` at my end, currently using the one created by you, it shouldn't be like that.
kindly add these two scripts `build` and `development`